### PR TITLE
Implement Webhook Tracking API

### PR DESCRIPTION
**Pull request by maxyengine**

### CODE REVIEW

File: src/Entities/Schema.php [hunk 1]
```diff
@@ -195,6 +195,15 @@ public function __construct()
             '3dsecure/{3dsecureId}' => function (array $content) {
                 return new ThreeDSecure($content);
             },
+            'reporting-currencies' => function (array $content) {
```
Comment (slavcodev) [inline src/Entities/Schema.php]: Please remove this unnecessary builder, was added after conflicts resolving.

File: src/Services/WebhookTrackingService.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    /**
+     * @param string $logId
+     * @param array|ArrayObject $params
+     *
+     * @throws NotFoundException The resource data does exist
+     *
+     * @return WebhookTracking
+     */
+    public function load($logId, $params = [])
```
Comment (slavcodev) [inline src/Services/WebhookTrackingService.php]: Maybe rename argument to `trackId`?

File: src/Services/WebhookTrackingService.php [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    {
+        return $this->client()->get('tracking/webhooks/{logId}', ['logId' => $logId] + (array)$params);
```
Comment (slavcodev) [inline src/Services/WebhookTrackingService.php]: Wrong CS, space after casting `(array) `

**Comment by maxyengine** [comment]
test please

**Comment by maxyengine** [comment]
retest please

**Pull request merged**